lib/wallaroo/core/source/gen_source/gen_source_listener.pony

```diff
@@ -140,6 +144,17 @@ actor GenSourceListener[In: Any val] is SourceListener
     end
     _sources.push(source)
 
+  be start_listening() =>
```

Comment:
I think we should be removing `start_listening` and `start_sources` behavior from `SourceListener` and all instances now that we're handling this directly in the application init lifecycle phases.